src/components/atoms/SfCircleButton/SfCircleButton.html

```diff
@@ -0,0 +1,3 @@
+<SfButton class="sf-circle-button">
+    <img class="sf-circle-button__icon" :src="icon" :alt="alt"/>
```

Comment:
add a default slot here too
```suggestion
    <slot><img class="sf-circle-button__icon" :src="icon" :alt="alt"/></slot>
```